orttraining/orttraining/eager/test/ort_ops.py

```diff
@@ -8,6 +8,55 @@
 import numpy as np
 import onnxruntime_pybind11_state as torch_ort
 import torch
+from parameterized import parameterized
+
+# OPS - is a list of  list of [test_operator, the tested_tensor]
+ops = [
+    ["abs", torch.tensor([-1, -2, 3, -6, -7])],
+    ["acos"],
+    ["acosh", torch.rand(6).uniform_(1, 9999)],  # limits[1,INF]
+    ["asinh"],
+    ["atanh", torch.rand(6).uniform_(-1, 1)],
+    ["asin", torch.rand(6).uniform_(-1, 1)],
+    ["atan"],
+    ["ceil"],
+    ["cos"],
+    ["cosh"],
+    ["erf"],
+    ["exp"],
+    ["floor"],
+    ["log", torch.abs(torch.rand(6)) + 0.5],
+    ["reciprocal"],
+    ["neg", torch.randn(10)],
+    ["round"],
+    ["relu"],
+    ["sigmoid"],
+    ["sin"],
+    ["sinh"],
+    ["sqrt", torch.randn(10).uniform_(1, 9999)],
```

Comment:
looks like uniform_ will completely replace randn. Are you using randn to create a tensor of size 10? Also why use uniform_ for sqrt and not just rand to abs(rand) (the latter if nan is the issue)?